### ISSUE TITLE
No sync works after updating to 0.10.0

### ISSUE THREAD

**Issue by vitezfh** [MrAlucardDante/baikal-docker-hass#12]
My compose.yaml:
```
services:
  baikal:
    labels:
      traefik.enable: true
      traefik.http.routers.baikal.rule: Host(`cal.vitez.xyz`)
      traefik.http.routers.baikal.tls.certresolver: acmeresolver
      traefik.http.routers.baikal.entrypoints: websecure
      traefik.http.routers.baikal.middlewares: my-geoblock@file
    image: 'ghcr.io/mralucarddante/baikal-docker-hass:0.10.0'
    volumes:
      - /docker-data/baikal/config:/var/www/baikal/config
      - /docker-data/baikal/data:/var/www/baikal/Specific
    networks:
       - traefik-net
    restart: unless-stopped
    environment:
      - TZ=Europe/Berlin
networks:
  traefik-net:
      external: true
```
Logs:
```
baikal-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
baikal-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
baikal-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
baikal-1  | 10-listen-on-ipv6-by-default.sh: info: /etc/nginx/conf.d/default.conf differs from the packaged version
baikal-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-create-baikal-database-folder.sh
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/40-disable-nginx-ipv6-if-unsupported.sh
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/40-fix-baikal-file-permissions.sh
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/40-php-fpm.sh
baikal-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/50-start-msmtp.sh
baikal-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: using the "epoll" event method
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: nginx/1.27.2
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: built by gcc 12.2.0 (Debian 12.2.0-14) 
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: OS: Linux 6.6.62-1-lts
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: getrlimit(RLIMIT_NOFILE): 1024:524288
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: start worker processes
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: start worker process 50
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: start worker process 51
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: start worker process 52
baikal-1  | 2024/11/18 16:50:49 [notice] 1#1: start worker process 53
baikal-1  | 172.22.0.13 - - [18/Nov/2024:16:57:21 +0100] "REPORT /dav.php/calendars/calendar/default/ HTTP/1.1" 302 5 "-" "Mozilla/5.0" "84.152.220.209"
baikal-1  | 172.22.0.13 - - [18/Nov/2024:16:57:21 +0100] "GET /dav.php/calendars/calendar/default/admin/install/ HTTP/1.1" 302 5 "-" "Mozilla/5.0" "84.152.220.209"
baikal-1  | 172.22.0.13 - - [18/Nov/2024:16:57:21 +0100] "GET /dav.php/calendars/calendar/default/admin/install/admin/install/ HTTP/1.1" 302 5 "-" "Mozilla/5.0" "xxx.xxx.xxx.xxx"
```
This keeps spamming/escalating to ever longer variations of:
`baikal-1  | 172.22.0.13 - - [18/Nov/2024:17:13:03 +0100] "GET /dav.php/calendars/calendar/default/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/admin/install/ HTTP/1.1" 302 5 "-" "Mozilla/5.0" "xxx.xxx.xxx.xxx"`

No sync works after updating to 0.10.0
I reverted to 0.9.5 and it's fine again.

**Comment by vitezfh** [comment]
FYI I haven't found any github issues regarding these problems in baikal-docker repo or the actual baikal repo.

**Comment by MrAlucardDante** [comment]
Hi. I just checked my logs and I don't have anything like this. I have 2 users : one on iOS connected natively and one on Android connected with DAVx5 and baikal had been running for 4 days on 0.10.1. 

Keep in mind that my image is basically ckulka's image with a fix in one php file for home assistant. So any issues should at least be reported to them or the baikal. 

It looks to me like a traefik issue though. Sounds like a recursive call on admin/install more than an actual baikal issue.

**Comment by starsoccer** [comment]
Just in case it helps others it does seem to be some sort of routing issue. Im having the same issue switching mine and I am unclear as to why. I changed the containers and for some reason the local DNS I have wont resolve properly through Nginx Proxy Manager even though the container and static ips are all the same.

Edit: After more testing my side, the issue was due to the fact that I was forwarding traffic to port 443 as https but this image does not have port 443 open and I needed to use 80 instead and http
